sources/engine/Stride.Engine/Rendering/Compositing/ForwardRenderer.cs

```diff
@@ -472,7 +472,9 @@ protected static PixelFormat ComputeNonMSAADepthFormat(PixelFormat format)
         private void ResolveMSAA(RenderDrawContext drawContext)
         {
             // Resolve render targets
-            currentRenderTargetsNonMSAA.Resize(currentRenderTargets.Count, false);
+            currentRenderTargetsNonMSAA.Clear();
+            currentRenderTargetsNonMSAA.EnsureCapacity(currentRenderTargets.Count);
+            
```

Comment:
This isn't quite the same, right now, the way you have it setup is that you're **clearing the whole list** and then making sure the list can contain **at least** `currentRenderTargets.Count` long.
Resize changes the list's capacity to hold the given amount, but also the list's length, so if it is 5 items long and you pass it 3, it will be 3 items long from then on, but it's capacity will still be 5 or whichever capacity the list had at the time.
And passing `false` to `Resize` only affects the list when decreasing its length, so when the list is 5 items in length and is changed to 3 items in length, the 4th and 5th item will be set to null/default.